### ISSUE TITLE
No focus style on "Show on map" link, place profile page

### ISSUE THREAD

**Issue by smcmurtry** [cds-snc/covid-alert-portal#415]
This should have a yellow focus style similar to other anchor elements, when focus is moved here using the tab key:
<img width="843" alt="Screen Shot 2021-03-08 at 2 30 52 PM" src="https://user-images.githubusercontent.com/5498428/110384635-29482b80-801b-11eb-8c5b-7ac116b4b0c7.png">
